events_emitter/tasks.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import
+import logging
+
+
+from celery import group, shared_task, chain
+from datetime import datetime, timedelta
+from django.conf import settings
+
+from events_emitter.models import BusinessRules, EventsDependencies
+from events_emitter.utils import parse_str_to_timedelta, fire_action_pubsub
+from events_emitter.events_factory.factory_event import FactoryEvent
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+"""
+                          -                 -
+                         -- proccess rule 1 --
+                        ---                 ---
+Get business rules --> ---- proccess rule 2 ---- --> evaluate rules result with exprissions
+                        ---                 ---
+                         -- proccess rule 3 --
+                          -                 -
+"""
+@shared_task
+def proccess_business_rules(*args, **kwargs):
+    rules = BusinessRules.objects.all() # noqa
+    logger.info("start proccess business rules")
+    proccess_rule_list = []
+    for rule in rules:
+        logger.info(f"business rule {rule.event_type} with duration {rule.duration} and state {rule.state}")
+        curr_rule = {
+            'id': rule.id,
+            'event_type': rule.event_type,
+            'duration': str(rule.duration),
+            'state': rule.state
+        }
+        proccess_rule_list.append(proccess_rule.s(curr_rule))
+
+    proccess_rules_group = chain(group(proccess_rule_list), proccess_rules_res.s())
+    proccess_rules_group.apply_async(queue=settings.EVENTS_EMITTER_QUEUE)
+
+
+@shared_task
+def proccess_rule(rule, *args, **kwargs):
+    logger.info(f"start evaluate rule {rule.get('event_type')}")
+    rule_res = False
+    try:
+        time_series_type = FactoryEvent().create_event_class(settings.TIME_SERIES_TYPE)
```

Comment:
can we change this variable name into something more meaningful eg "time_series_datastore"